packages/terra-core-docs/src/terra-dev-site/doc/switch/AccessibilityGuide.2.doc.mdx

```diff
@@ -0,0 +1,179 @@
+import { Notice } from "@cerner/terra-docs";
+import { Badge } from 'terra-image/package.json?dev-site-package';
+import Whitespace from "../../common/layout-helpers/Whitespace";
+import SwitchLabel from './example/SwitchLabel';
+import AllergiesSwitch from './example/AllergiesSwitch';
+
+<Badge />
+
+# Accessibility Guide for Terra Switch
+
+## Why is the accessibility of Terra Switch? 
+
+The accessibility of a switch is imperative because a switch is a control users use to turn a setting on or off. If switches are not accessible, it can prevent users from understanding what state the switch is in, or worse, prevent the users from interacting with the product. By implemented switches in an accessible way, we are ensuring as many users can use the switch as possible regardless of how they may interact with it (e.g., with a mouse, touch, speech input, keyboard only, or other assistive technology).  
```

Comment:
```suggestion
The accessibility of a switch is imperative because a switch is a control users use to turn a setting on or off. If switches are not accessible, it can prevent users from understanding what state the switch is in, or worse, prevent the users from interacting with the product. By implementing switches in an accessible way, we are ensuring as many users can use the switch as possible regardless of how they may interact with it (e.g., with a mouse, touch, speech input, keyboard only, or other assistive technology).  
```